### PR TITLE
Adding "copyfiles" package

### PR DESCRIPTION
**Pull request by JonathanCrd**
Windows does not recognize the `cp src/generators/static/README.md.hbs dist/src/generators/static/README.md.hbs` command. The package [copyfiles](https://www.npmjs.com/package/copyfiles) fixes this and the command was changed to `copyfiles src/generators/static/README.md.hbs dist/src/generators/static/README.md.hbs`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
         "@azure/logger": "^1.0.0",
         "@opentelemetry/api": "^0.10.2",
         "@types/lodash": "^4.14.149",
+        "copyfiles": "^2.4.1",
```
Comment (deyaaeldeen) [inline package.json:48]: it is not needed here.

**Review by deyaaeldeen — COMMENTED** [review]
Congrats on your first contribution! That was quick! 

I left one comment.

File: package.json (hunk 1 above)
Comment (JonathanCrd) [reply package.json]: Got it, it's been removed

**Review by joheredi — APPROVED** [review]
Looks great! Thanks for fixing this 😄

**Review by deyaaeldeen — APPROVED** [review]

**Review by sarangan12 — APPROVED** [review]
Looks good

**Pull request merged**